operator/config/default/kustomization.yaml

```diff
@@ -11,9 +11,11 @@ namePrefix: module-manager-
 # Labels to add to all resources and selectors.
 commonLabels:
   app.kubernetes.io/component: module-manager.kyma-project.io
-
-configurations:
-  - ./commonlabels_override.yaml
+  app.kubernetes.io/instance: module-manager-main
```

Comment:
```suggestion
  app.kubernetes.io/instance: main
```

Reply:
If go this way, we should also align in lifecycle-manager